### PR TITLE
fix emscripten build

### PR DESCRIPTION
**Pull request by midzer**
Hi,

I wondered why I can't find an Emscripten port online, so I fixed the shell script to release it myself.

Have a nice day
midzer

### CODE REVIEW

File: make_emscripten.sh [hunk 1]
```diff
@@ -4,35 +4,45 @@ set -e
 rm -rf emscripten/*
 mkdir -p emscripten
 
-cmake .
```
Comment (cxong) [inline make_emscripten.sh:7]: I think this is required, why remove it?

File: make_emscripten.sh [hunk 2]
```diff
@@ -4,35 +4,45 @@ set -e
...
-    --preload-file doc \
-    --preload-file dogfights \
-    --preload-file graphics \
-    --preload-file missions \
-    --preload-file music \
-    --preload-file sounds \
-    -o emscripten/index.html
+    -s USE_OGG=1
+
+emcc -O3 \
```
Comment (cxong) [inline make_emscripten.sh:28]: why build twice?

Comment (midzer) [reply make_emscripten.sh:28]: First step is compile, second is link.

File: make_emscripten.sh (hunk 1 above)
Comment (midzer) [reply make_emscripten.sh:7]: It was not necessary (for me).

Comment (cxong) [reply make_emscripten.sh:7]: I think we need to run cmake to generate some files - you might have run it before

Comment (midzer) [reply make_emscripten.sh:7]: Yeah, thats possible. Let me update this PR.

**Pull request merged**